src/utils.js

```diff
@@ -10,10 +12,19 @@ module.exports = {
    return !!(object && object.constructor && object.call && object.apply);
   },
 
-  doCallback: function(hook, config, arg) {
+  doCallback: function(hook, config, arg, deprecated) {
     if(this.isFunction(config.callbacks[hook])) {
       config.callbacks[hook](arg);
+      if (deprecated) this.logDeprecated(hook + ' callback has been replaced, please see docs')
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '{' and instead saw 'this'.](https://www.codacy.com/app/laander/booking-js/file/2061450308/issues/source?bid=2969108&fileBranchId=3135617#l18)